taxi_service/urls.py

```diff
@@ -22,4 +22,7 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("taxi.urls", namespace="taxi")),
+    path("accounts/", include("django.contrib.auth.urls")),
+
+    path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
delete extra empty line

Reply:
I did it for purpose for better readability, to separate debug toolbar. Is it bad practice?